test/compact.jl

```diff
@@ -118,15 +118,6 @@ end
     @test similar_strings(get_model_string(model), expected_string)
   end
 
```

Comment:
The old test can be adapted:
```suggestion
  @testset "Custom naming of @compact with NoShow" begin
    _model = @compact(w=Dense(32, 32)) do x, y
      tmp = sum(w(x))
      return tmp + y
    end
    model = NoShow("Linear(...)", _model) 
    expected_string = "Linear(...)         # 1_056 parameters"
    @test similar_strings(get_model_string(model), expected_string)
  end
```
Might be better for it to live in a `test/noshow.jl` file too, although here we get to reuse `similar_strings` / `get_model_string`

Reply:
I put this here, and other tests in test/noshow.jl